packages/react/package.json

```diff
@@ -22,28 +22,27 @@
   },
   "license": "MIT",
   "author": "Clerk",
-  "sideEffects": [
-    "./dist/cjs/index.js",
-    "./dist/cjs/polyfills.js",
-    "./dist/esm/index.js",
-    "./dist/esm/polyfills.js"
-  ],
   "exports": {
+    "./package.json": "./package.json",
```

Comment:
I didn't find any source for the reason but from what I've seen the convention is to put this line to the end of the exports map, not at the beginning, e.g. https://reactnative.dev/blog/2023/06/21/package-exports-support#what-is-package-exports

Since this is a workaround for some bundlers and it resolves things from top to bottom it makes sense to put this to the bottom

Reply:
Export order updated.